src/open_inwoner/templates/pages/cases/list_inner.html

```diff
@@ -16,38 +16,42 @@ <h1 class="utrecht-heading-1" id="cases">{{ page_title }} ({{ paginator.count }}
             <div class="card__body">
                 <!-- submission cases -->
                 {% if case.case_type == "OpenSubmission" %}
-                    <a href="{{ case.vervolg_link }}"  class="cases__link">
-                    <p class="h4"><span class="link link__text">{{ case.naam }}</span></p>
+                    <a href="{{ case.vervolg_link }}" class="cases__link">
+                    <h2 class="card__heading--h4"><span class="link link__text">{{ case.naam }}</span></h2>
```

Comment:
Maybe remove the `h4` from the class and just use a descriptive class name: `car__heading`, `card__date-caption`, or sth. like this.

Reply:
@pi-sigma Yes, it looks like in the new design ALL headings are going to be 18px, so I'll need to make this a generic heading class, mostly set on H2.